app/src/main/java/org/schabi/newpipe/ui/components/items/stream/StreamListItem.kt

```diff
@@ -0,0 +1,85 @@
+package org.schabi.newpipe.ui.components.items.stream
+
+import android.content.res.Configuration
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.combinedClickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import org.schabi.newpipe.extractor.stream.StreamInfoItem
+import org.schabi.newpipe.ui.theme.AppTheme
+
+@OptIn(ExperimentalFoundationApi::class)
```

Comment:
Is it possible to use an element that doesn't require the ExperimentalFoundationApi optin? I realize it may not be possible, but if possible, I believe it reduces the risk of new bugs being introduced if the underlying element is removed from the library or fundamentally changed. I realize that I'm not as familiar with this code base, so the process here may be different, so please don't hesitate to disregard this comment. I just wanted to weigh in and find out your thoughts.

Reply:
That annotation is needed because the `combinedClickable` modifier is currently experimental.